controls/srg_gpos/SRG-OS-000358-GPOS-00145.yml

```diff
@@ -1,10 +1,21 @@
 controls:
     -   id: SRG-OS-000358-GPOS-00145
         levels:
-            - medium
+            - low
         title: {{{ full_name }}} must record time stamps for audit records that meet
             a minimum granularity of one second for a minimum degree of precision.
-        rules:
-            - audit_rules_suid_privilege_function
-            - package_audit_installed
-        status: automated
```

Comment:
latest RHEL8 STIG has package audit and service enabled to cover this requirement. We can probably use them here instead of `inherently met`

Reply:
So the RHEL8 STIG has this both as "Applicable - Configurable" for the package install and "Inherently met" with this text. Are we losing information we just add it as an SRG to the package install?

Reply:
I'm thinking the answer is "Inherently met". Those services and packages help with the audit, but this SRG is specifically talking about how the operating system records audit records with time down to the second. Which that is "Inherently met".